src/optimizer/properties.cpp

```diff
@@ -83,7 +83,7 @@ std::string PropertyColumns::ToString() const {
 PropertyDistinct::PropertyDistinct(std::vector<
     std::shared_ptr<expression::AbstractExpression>> distinct_column_exprs)
     : distinct_column_exprs_(std::move(distinct_column_exprs)) {
-  LOG_TRACE("Size of column property: %lu", GetSize());
+  //LOG_TRACE("Size of column property: %ld", columns_.size());
 }
```

Comment:
This used to fail compilation in TRACE logging mode, but it's been fixed in master.